__tests__/ButtonNavigation.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { mount } from '@vue/test-utils'
+
+import ButtonNavigation from '../src/components/ButtonNavigation.vue'
+
+describe('header components', () => {
+    it('get nav', () => {
```

Comment:
kalau pakai `it` di deskripsinya tambahin `should` mas, misal `should get nav`

Reply:
oke mas